src/workerd/util/autogate.c++

```diff
@@ -9,6 +9,8 @@
 #include <kj/common.h>
 #include <kj/debug.h>
 
+#include <cstdlib>
```

Comment:
Nitpick: Please use `stdlib.h`.

The entire purpose of `cstdlib` is to provide the symbols of `stdlib.h`, except declared in the `std::` namespace -- in order to avoid polluting the global namespace. But in practice it doesn't do this. In your code you called `getenv()` instead of `std::getenv()`, and it still worked. This is because all compilers have decided to have `cstdlib` (and similar headers) put the symbols in _both_ locations. But this defeats the purpose.

So KJ style is to just use `stdlib.h` and not expect symbols to be in the `std::` namespace.

Reply:
TIL. Thanks!